### DISCUSSION TITLE
Wrap the C SDK or create native Swift

### DISCUSSION THREAD

**Discussion by dangdennis** [sqlitecloud/discussions#3] [SDK] [answered]
Heya, I'm experimenting with writing a swift client myself. I noticed that instead of wrapping the C SDK, you chose to create a native Go client. As far as I can see, I don't see any cgo being used.

Is there a reason why you chose not to wrap the C client? I'm debating whether to continue tinkering with the Swift build tool versus just writing it all natively.

**Comment by dangdennis** [comment, marked as answer]
Nevermind! I'm blind. There's already a client https://github.com/sqlitecloud/swift.

**Comment by marcobambini** [comment]
Hi @dangdennis please note that we need to update the C SDK used by the Swift SDK.

**Reply by dangdennis** [reply]
Thanks for the heads up. I plan to test it soon, but can't tell if it also supports swift on linux. Will drop an issue later there.
